blocks/rich-text/index.js

```diff
@@ -884,6 +933,7 @@ export default class RichText extends Component {
 					{ ...ariaProps }
 					className={ className }
 					key={ key }
+					onMount={ this.onTinyMCEMount }
```

Comment:
Should we move these scrolling behaviors to the TinyMCE component instead of adding this Prop?

Reply:
I did, but then we need to pass the `scrollPosition` variable which is tied to this component. To me it looked cleaner this way, but I'm fine either way.

Reply:
True, I missed the global variable

Reply:
Global sounds bad :) I'd prefer to call it module variable ;)